package.json

```diff
@@ -29,5 +29,8 @@
   "license": "MIT",
   "devDependencies": {
     "@microsoft/rush": "4.3.0"
+  },
+  "dependencies": {
+    "npm": "^6.0.0"
```

Comment:
This is actually unintentional.  npm told me at some point to run a command, which I diligently obeyed, and this edit got auto-added.  Commentary is welcome on why npm thought this was necessary... ?

Reply:
I believe npm was telling you that a new version of npm is available and prompting you to upgrade :) But.. you might have entered incorrectly (specifically, missed the `-g` option so it became a local install that was written to package.json)

You probably want to revert this.